sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py

```diff
@@ -196,6 +196,22 @@ def __init__(
         self._client._config.version = get_api_version(kwargs)  # type: ignore [assignment]
         self._configure_encryption(kwargs)
 
+    async def __aenter__(self):
+        await self._client.__aenter__()
+        return self
+
+    async def __aexit__(self, *args):
+        await self._client.__aexit__(*args)
+
+    async def close(self) -> None:
+        """This method is to close the sockets opened by the client.
+        It need not be used when using with a context manager.
+
+        :return: None
+        :rtype: None
```

Comment:
Can we be consistent with the docs between sync/async? Here we document the return type, in sync we don't. We probably don't need to too IMO but it's up to you.